PowerEditor/installer/nativeLang/italian.xml

```diff
@@ -1160,7 +1160,19 @@ Puoi indicare serie di delimitatori di colonna usando degli spazi per separare i
 (non selezionarlo se non sai cosa stai facendo)"/>
                 </Delimiter>
 
-                <Cloud title="Cloud e link">
+                <Performance title="Prestazioni">
+                    <Item id="7141" name="Restrizioni per File di grandi dimensioni"/>
+                    <Item id="7143" name="Abilita restrizioni per i file di grandi dimensioni (no evidenziazione sintattica"/>
```

Comment:
(no evidenziazione sintattica_

Bisogna la parentesi chiusa

Reply:
@rdipardo, thanks, done. Do you agree with the others modifications? Are they better than the previous translation?

Reply:
> @rdipardo, thanks, done. Do you agree with the others modifications? Are they better than the previous translation?

Tutto bene per quanto ci vedo. Buon lavoro!